src/pages/OnboardingAccounting/BaseOnboardingAccounting.tsx

```diff
@@ -59,14 +61,15 @@ function BaseOnboardingAccounting({shouldUseNativeStyles, route}: BaseOnboarding
     // If the signupQualifier is VSB, the company size step is skip.
     // So we need to create the new workspace in the accounting step
     useEffect(() => {
-        if (!isVsb || !!onboardingPolicyID || isLoadingOnyxValue(onboardingPolicyIDResult)) {
+        const filteredPolicies = Object.values(allPolicies ?? {}).filter(PolicyUtils.isPaidGroupPolicy);
```

Comment:
Lets change this to `some`, we just need to know they have one paid policy we dont have to iterate over the entire list

Reply:
@mountiny ^

Reply:
You can just add it in some future pr

Reply:
I can raise a quick one now and you can merge it. WDYT??